python/sglang/srt/utils/common.py

```diff
@@ -3618,6 +3618,12 @@ def my_kernel(x_ptr, y_ptr, BLOCK_SIZE: tl.constexpr):
     """
 
     def decorator(fn):
+        # Auto-enable the custom kernel cache for CUDA, where it is
+        # known to be compatible.
+        if is_cuda() and not envs.SGLANG_USE_CUSTOM_TRITON_KERNEL_CACHE.get():
+            logger.debug("Detected platform CUDA, using custom triton kernel cache.")
+            return CachedKernel(fn, key_fn)
```

Comment:
I think the expected logic should be:

enable custom cache when either:
1. SGLANG_USE_CUSTOM_TRITON_KERNEL_CACHE not specified **AND** is_cuda == True, 
2. SGLANG_USE_CUSTOM_TRITON_KERNEL_CACHE is true

Can you help double check if your code correctly handles the case when CUDA is true but SGLANG_USE_CUSTOM_TRITON_KERNEL_CACHE  is set to false?